packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -1806,27 +1807,6 @@ describe('internal-plugin-metrics', () => {
         });
       });
 
-      it('should override custom properties for an unknown error', () => {
```

Comment:
why is this test removed?

Reply:
Same as above,
`This was added recently into beta. We have cherry-picked the commit only today morning. https://github.com/webex/webex-js-sdk/pull/3525. WIll update the branch.`